pkg/plugin/trivy/plugin.go

```diff
@@ -946,3 +980,43 @@ func GetScoreFromCVSS(CVSSs map[string]*CVSS) *float64 {
 
 	return nvdScore
 }
+
+func fullImagePath(image string) string {
+	const defaultRegistry = "index.docker.io"
+	registryAndImage := strings.Split(image, "/")
```

Comment:
You don't have to parse image reference manually. We're using github.com/google/go-containerregistry library in Starboard.
Beyond that, we already have a similar method called `parseImageRef` so maybe you can reuse somehow.

Reply:
Great Idea. I updated the PR and it is now using the library github.com/google/go-containerregistry to parse and expand the image. It is actually much easier.